lib/modules/datasource/postprocess-release.ts

```diff
@@ -0,0 +1,37 @@
+import { logger } from '../../logger';
+import { getDatasourceFor } from './common';
+import type { PostprocessReleaseConfig, Release } from './types';
+
+export async function postprocessRelease(
+  config: PostprocessReleaseConfig,
+  release: Release,
+): Promise<Release | null> {
+  const { datasource } = config;
+
+  if (!datasource) {
+    return release;
+  }
+
+  const ds = getDatasourceFor(datasource);
+  if (!ds) {
+    return release;
```

Comment:
warn too, should never happen

Reply:
Actually this is the most common path - most datasources don't have this function